roles/nginx/templates/nginx.conf.j2

```diff
@@ -1,57 +1,60 @@
 # {{ ansible_managed }}
 
-# nginx Configuration File
-# http://wiki.nginx.org/Configuration
+# Configuration File - Nginx Server Configs
+# http://nginx.org/en/docs/dirindex.html
 
 {% block modules_enabled -%}
 include modules-enabled/*.conf;
 {% endblock %}
 
 {% block user %}
-# Run as a less privileged user for security reasons.
+# Run as a unique, less privileged user for security reasons.
+# Default: nobody nobody
 user {{ nginx_user }};
 {% endblock %}
 
 {% block worker %}
-# How many worker threads to run;
-# "auto" sets it to the number of CPU cores available in the system, and
-# offers the best performance. Don't set it higher than the number of CPU
-# cores if changing this parameter.
-
-# The maximum number of connections for Nginx is calculated by:
-# max_clients = worker_processes * worker_connections
+# Sets the worker threads to the number of CPU cores available in the system for best performance.
+# Should be > the number of CPU cores.
+# Maximum number of connections = worker_processes * worker_connections
+# Default: 1
 worker_processes auto;
 
-# Maximum open file descriptors per process;
-# should be > worker_connections.
+# Maximum number of open files per worker process.
+# Should be > worker_connections.
+# Default: no limit
 worker_rlimit_nofile 8192;
 {% endblock %}
 
 {% block events %}
 events {
-  # When you need > 8000 * cpu_cores connections, you start optimizing your OS,
-  # and this is probably the point at which you hire people who are smarter than
-  # you, as this is *a lot* of requests.
+  # If you need more connections than this, you start optimizing your OS.
+  # That's probably the point at which you hire people who are smarter than you as this is *a lot* of requests.
+  # Should be < worker_rlimit_nofile.
+  # Default: 512
   worker_connections 8000;
 }
 {% endblock %}
 
 {% block error_log %}
-# Default error log file
-# (this is only used when you don't override error_log on a server{} level)
+# Log errors and warnings to this file
+# This is only used when you don't override it on a server{} level
+# Default: logs/error.log error
 error_log  {{ nginx_logs_root }}/error.log warn;
 {% endblock %}
 
 {% block pid %}
 # The file storing the process ID of the main process
-pid        /run/nginx.pid;
+# Default: nginx.pid
+pid        /var/run/nginx.pid;
```

Comment:
Was this change from h5bp? I believe it should stay as `/run`. It's the "newer" location and `/var/run` is symlinked to it.

Reply:
it's from h5bp:
https://github.com/h5bp/server-configs-nginx/blob/c5c6602232e0976d9e69d69874aa84d2a2698265/nginx.conf#L34

Reply:
So `/run` is the correct one?

Reply:
From what I can tell it is. So let's keep it as `/run`

Reply:
reverted